services/docker-compose-build.yml

```diff
@@ -249,3 +249,26 @@ services:
         org.label-schema.vcs-url: "${VCS_URL}"
         org.label-schema.vcs-ref: "${VCS_REF}"
         io.osparc.api-version: "${STORAGE_API_VERSION}"
+
+  agent:
+    image: local/agent:${BUILD_TARGET:?build_target_required}
+    build:
+      context: ../
+      dockerfile: services/agent/Dockerfile
+      args:
+        - tag=latest
+        - VCS_REF=${VCS_REF}
+        - VCS_REF_CLIENT=${VCS_REF_CLIENT}
+        - VCS_STATUS_CLIENT=${VCS_STATUS_CLIENT}
+        - VCS_URL=${VCS_URL}
+      cache_from:
+        - local/agent:${BUILD_TARGET:?build_target_required}
+        - ${DOCKER_REGISTRY:-itisfoundation}/agent:master-github-latest
+        - ${DOCKER_REGISTRY:-itisfoundation}/agent:staging-github-latest
+        - ${DOCKER_REGISTRY:-itisfoundation}/agent:release-github-latest
+      target: ${BUILD_TARGET:?build_target_required}
+      labels:
+        org.label-schema.schema-version: "1.0"
+        org.label-schema.build-date: "${BUILD_DATE}"
+        org.label-schema.vcs-url: "${VCS_URL}"
+        org.label-schema.vcs-ref: "${VCS_REF}"
```

Comment:
since there is actually an API, maybe it should have a version too like the others above?

Reply:
Uhm, not actually an API, just has none versione endpoint for health

Reply:
@sanderegg means the version you have in ``VERSION`` file. In this repo service and API (if any) version are the same. 

SEE who we add that in the image labels: https://github.com/ITISFoundation/osparc-simcore/blob/3f73cdc7e126c0878aa2a4b6f06159d0793aa171/Makefile#L57